public/src/main/java/com/revenuecat/purchases/DangerousSettings.kt

```diff
@@ -9,5 +9,7 @@ data class DangerousSettings(
      * automatically, and you will have to call syncPurchases whenever a new purchase is completed in order to send it
      * to the RevenueCat's backend. Auto syncing of purchases is enabled by default.
      */
-    val autoSyncPurchases: Boolean = true
+    val autoSyncPurchases: Boolean = true,
+
+    val useBC5: Boolean = false
```

Comment:
this wouldn't necessary have to be a dangerous setting, could just be part of the builder directly

Reply:
In the end the problem also is, how much complexity do we push down to our users... to me it sounds like they shouldn't really care too much whether we're using BC4 or BC5, so the concern is that we're adding extra complexity to them using our SDK unnecessarily